glambie/data/timeseries.py

```diff
@@ -280,10 +284,21 @@ def convert_timeseries_to_unit_mwe(self, density_of_water: float = constants.DEN
         else:
             object_copy = copy.deepcopy(self)
             object_copy.unit = "mwe"
-            if self.unit == "m":  # @TODO: convert uncertainties as well
+            if self.unit == "m":
                 object_copy.data.changes = np.array(meters_to_meters_water_equivalent(object_copy.data.changes,
                                                                                       density_of_water=density_of_water,
                                                                                       density_of_ice=density_of_ice))
+                # Uncertainties
+                # First, convert elevation change error in m to mwe
+                object_copy.data.errors = np.array(meters_to_meters_water_equivalent(object_copy.data.errors,
+                                                                                     density_of_water=density_of_water,
+                                                                                     density_of_ice=density_of_ice))
+                # Second, include density uncertainty in error
+                density_unc = get_density_uncertainty_over_survey_period(self.data.max_temporal_resolution)
+                df = object_copy.data.as_dataframe()
+                # also see formula in Glambie Assessment Algorithm document, section 5.2 Homogenization of data
```

Comment:
This is good, this is what I was looking for last time. Provided it's easy to trace back and forth, it's usually easy to spot implementation issues.